applications/models.py

```diff
@@ -203,11 +209,14 @@ def add_resume(self, *, resume_file=None, linkedin_url=None):
     )
     def approve_submission(self):
         """Approve application submission"""
-        return (
-            AppStates.AWAITING_PAYMENT.value
-            if self.is_ready_for_payment()
-            else AppStates.AWAITING_USER_SUBMISSIONS.value
-        )
+        if self.is_ready_for_payment():
+            try:
+                create_and_send_applicant_letter(application=self, is_acceptance=True)
+            except:
+                log.exception(f"Unable to send applicant approval letter for {self}")
```

Comment:
`cms.models.LetterTemplatePage.DoesNotExist: LetterTemplatePage matching query does not exist` : still not very familiar with wagtail, do I need to manually create this template?

Reply:
Yes, sorry I haven't written the testing instructions yet. You need to create it as a child page of the home page in wagtail.

Reply:
thanks, got it working, the emails look good

Reply:
You should also see the letter that was sent by going to `/letters/uuid-goes-here/`. You will need to look up the hash uuid value in `ApplicantLetter`, and also include the dashes in the URL